fab/services/templates/supervisor_celery_repeat_record_queue.conf

```diff
@@ -1,6 +1,6 @@
 [program:{{ project }}-{{ environment }}-celery_repeat_record_queue]
 environment={% for name, value in supervisor_env_vars.items %}{{ name }}="{{ value }}"{% if not forloop.last %},{% endif %}{% endfor %}
-command={{ new_relic_command }}{{ virtualenv_current }}/bin/python {{ code_current }}/manage.py celery worker --queues=repeat_record_queue --events --loglevel=INFO --hostname={{ host_string }}_repeat_record_queue --maxtasksperchild=500 --concurrency={{ celery_params.concurrency }} -Ofair
+command={{ new_relic_command }}{{ virtualenv_current }}/bin/python {{ code_current }}/manage.py celery worker --queues=repeat_record_queue --events --loglevel=INFO --hostname={{ host_string }}_repeat_record_queue --maxtasksperchild=500 -P gevent --concurrency={{ celery_params.concurrency }} -Ofair
```

Comment:
we can get rid of the maxtasksperchild setting since that doesn't apply to gevent pooling, and we can also remove -Ofair (i'm not sure if that applies to gevent pooling, but we wouldn't mind prefetching tasks for this situation anyway since the tasks run so fast so it's fine to remove)